libcst/_parser/_grammar.py

```diff
@@ -309,22 +314,74 @@ def validate_grammar() -> None:
                 )
 
 
-def get_nonterminal_productions() -> Iterator[Production]:
+def _get_version_comparison(version: str) -> Tuple[str, PythonVersionInfo]:
+    if version.startswith(">="):
+        return (">=", parse_version_string(version[2:].strip()))
+    if version.startswith("<="):
+        return ("<=", parse_version_string(version[2:].strip()))
+    if version.startswith("=="):
+        return ("==", parse_version_string(version[2:].strip()))
+    if version.startswith("!="):
+        return ("!=", parse_version_string(version[2:].strip()))
+    if version.startswith(">"):
+        return (">", parse_version_string(version[1:].strip()))
+    if version.startswith("<"):
+        return ("<", parse_version_string(version[1:].strip()))
```

Comment:
Those 10 lines can become simpler as
``
for operator in [">=", "<=", "==", "!=", ">", "<"]:
    return return (operator, parse_version_string(version[len(operator):].strip()))
```